src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+public class Task {
+    private final String description;
+    private boolean isDone;
+    private static int num = 0;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        num++;
+    }
+
+    private String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public void markAsUndone() {
+        this.isDone = false;
+    }
+
+    public String getStatus() { return (isDone ? "1" : "0"); }
+
+    public String getDescription() { return description; }
+
+    public static int lsSize() { return num; }
```

Comment:
I'm not sure if I'm right but I believe this method serves to return the list size? If so, I think it would be a plus to take advantage of the naming convention as you have for other methods - i.e. getSize() 😄